utam-core/src/main/java/utam/core/element/Element.java

```diff
@@ -7,6 +7,7 @@
  */
 package utam.core.element;
 
+import java.time.Duration;
```

Comment:
redundant import i think

Reply:
This one I missed when I removed them. Fixed in next commit.